### PR TITLE
Fix travis build [#167300875]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 node_js:
-  - 6
+  - 8
 install:
   - travis_retry gem install s3_website -v 3.4.0
   - travis_retry pip install awscli --upgrade --user
-  - npm i -g yarn && yarn
+  - travis_retry npm ci
 before_script: webpack && npm test
 script: ./s3_deploy.sh
```
Comment (kswenson) [inline .travis.yml]: Following our recent conventions, this should really be:
```
script: webpack && npm test
after_succes: ./s3_deploy.sh
```
and while you're at it you should probably add `dist: trusty`.

**Review by scytacki — COMMENTED** [review]
This seems to remove references to yarn, but yarn is still in the package.json. Should it be removed from there too?

**Comment by dougmartin** [comment]
Yes, it should be - I didn't check as the line in the travis file installs it globally.  I'll remove it now.

**Comment by dougmartin** [comment]
@scytacki / @kswenson - I've updated the PR to remove all references to yarn and I've updated the travis config per @kswenson's recommendations.

**Review by scytacki — APPROVED** [review]
Looks great to me.

**Pull request merged**